src/wallet/wallet.cpp

```diff
@@ -2224,7 +2194,7 @@ CAmount CWallet::GetUnconfirmedWatchOnlyBalance() const
         {
             const CWalletTx* pcoin = &entry.second;
             if (!pcoin->IsTrusted() && pcoin->GetDepthInMainChain() == 0 && pcoin->InMempool())
-                nTotal += pcoin->GetAvailableWatchOnlyCredit();
+                nTotal += pcoin->GetAvailableCredit(true, ISMINE_WATCH_ONLY);
```

Comment:
nit: comment bool purpose?

Reply:
I don't think it's the project's coding style to always comment arguments (at least, I don't see that elsewhere in the codebase as the norm).